starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -52,7 +52,8 @@ public List<Inventory> findAll() {
    */
   public Inventory create(Inventory inventory) {
```

Comment:
How can you verify that the incoming inventory object is valid? What are the requirements for it to be valid?

Reply:
What if the inventory object being passed in is null? We don't want to get a null pointer exception.

Reply:
What you have as a null check is technically not wrong, but I was looking for something more like this: Assert.notNull(inventory, "Submitted Inventory item must not be null.");